.appveyor.yml

```diff
@@ -0,0 +1,35 @@
+# AppVeyor file
+# http://www.appveyor.com/docs/appveyor-yml
+
+# Build version format
+version: "{build}"
+
+# Scripts that are called at very beginning, before repo cloning
+init:
+  # Fix line endings on Windows
+  - git config --global core.autocrlf true
+
+# Test against this version of Node.js
+environment:
+  nodejs_version: "8"
+
+build: off
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install
+
+test_script:
+  # Output useful info for debugging.
+  - node --version
+  - npm --version
+  - npm run build
+  - npm run check:tests:ie
+
+notifications:
```

Comment:
@zenflow @toverux @acupajoe feel free to add your public email here if you want to get `on_build_status_changed` notifications from AppVeyor